rendermodules/materialize/schemas.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+
+from marshmallow import post_load
+
+import argschema
+from argschema.fields import (Str, OutputDir, Int, Boolean, Float,
+                              List, InputDir, InputFile, Dict, Nested)
+from rendermodules.module.schemas import RenderClientParameters
+
+
+class MaterializedBoxParameters(argschema.schemas.DefaultSchema):
+    stack = Str(required=True, description=(""))
+    rootDirectory = OutputDir(required=True, description=(""))
+    width = Int(required=True, description="")
+    height = Int(required=True, description="")
+    maxLevel = Int(required=False, default=0, description="")
+    fmt = Str(required=False)  # TODO validate to allow PNG, TIF(F?), JPG
+    maxOverviewWidthAndHeight = Int(required=False, description=(""))
+    skipInterpolation = Boolean(required=False, description=(""))
+    binaryMask = Boolean(required=False, description=(""))
```

Comment:
I think this description as a tuple will break the documentation

Reply:
not tuples, just empty strings.  Either way, I will need to commit to add documentation.